### PR TITLE
Tli 39 multiple choice true false

### PR DESCRIPTION
**Pull request by NVieregg**
Added the possibility to choose True/False with already populated answers in Choices component.

Please check if:

- [x] correct answers are populated
- [x] true/false are radio buttons
- [x] only one type is checked

### CODE REVIEW

**Comment by NVieregg** [comment]
@cmauf new commits pushed, please check!

File: src/plugins/laya-la-scmc/edit.vue [hunk 1]
```diff
@@ -279,23 +315,48 @@ Dependencies:
...
     </form>
+    <b-modal
+      id="confirm-change-tf"
+      :title="y18n('warning')"
+      header-bg-variant="danger"
+      ok-variant="danger"
+      :ok-title="y18n('ok')"
+      :cancel-title="y18n('cancel')"
+      centered
+      static-change-tf
```
Comment (cmauf) [inline src/plugins/laya-la-scmc/edit.vue:353]: this property has to be just `static`

File: src/plugins/laya-la-scmc/edit.vue [hunk 2]
```diff
@@ -78,6 +78,23 @@ Dependencies:
...
           </div>
+          <div class="form-check form-check-inline align-text-top">
+            <input
+              id="scmc-tf"
+              v-model="variation"
+              class="form-check-input"
+              type="radio"
+              name="tf"
+              :value="tf"
+              @click="$bvModal.show('confirm-change-tf')"
```
Comment (cmauf) [inline src/plugins/laya-la-scmc/edit.vue:89]: Modal shows, but if you click "cancel", the variation is still changed and you have uneditable input fields

File: src/plugins/laya-la-scmc/create.vue [hunk 3]
```diff
@@ -331,14 +392,32 @@ export default {
...
+          flagged: false,
+          id: uuidv4()
+        })
+        this.options.push({
+          text: this.y18n('layaLaScmc.edit.false'),
+          simple: this.y18n('layaLaScmc.edit.false'),
+          flagged: false,
+          id: uuidv4()
+        })
+      } if (this.variation === this.single || this.variation === this.multiple) {
```
Comment (cmauf) [inline src/plugins/laya-la-scmc/create.vue:412]: either insert `else` keyword or move `if` statement to new line

File: src/i18n/en.json [hunk 4]
```diff
@@ -378,6 +382,7 @@
 	"notifications.studentNewFlagAnswer.cta": "Click to see the Answer!",
 	"notifications.studentNewFlagAnswer.text": "You have a new answer to your question in Course <CID>.",
 	"notifications.studentNewFlagAnswer.title": "New clarification",
+	"ok": "Ok",
```
Comment (cmauf) [inline src/i18n/en.json:385]: `courseCreate.modal.ok` serves the purpose of a generic ok

**Review by cmauf — CHANGES_REQUESTED** [review]
Modal in edit.vue doesn't work as expected, additional minor issues

**Review by cmauf — APPROVED** [review]
ok

**Pull request merged**